analyses/molecular-subtyping-ATRT/03-ATRT-molecular-subtyping-gistic-comparison.Rmd

```diff
@@ -0,0 +1,148 @@
+---
+title: "ATRT Molecular Subtyping - GISTIC Comparison"
+output: 
+  html_notebook:
+    toc: TRUE
+    toc_float: TRUE
+author: Chante Bethell for ALSF CCDL
+date: 2019
+---
+
+This notebook addresses the issue of molecular subtyping ATRT samples. 
+
+# Usage
+
+This notebook is intended to be run via the command line from the top directory
+of the repository as follows:
+
+`Rscript -e "rmarkdown::render('analyses/molecular-subtyping-ATRT/03-ATRT-molecular-subtyping-gistic-comparison.Rmd', clean = TRUE)"`
+
+# Set Up
+
+```{r}
+# Get `magrittr` pipe
+`%>%` <- dplyr::`%>%`
+```
+
+## Directories and Files
+
+```{r}
+# Detect the ".git" folder -- this will in the project root directory.
+# Use this as the root directory to ensure proper sourcing of functions no
+# matter where this is called from
+root_dir <- rprojroot::find_root(rprojroot::has_dir(".git"))
+
+# Set path to results and plots directory
+results_dir <-
+  file.path(root_dir, "analyses", "molecular-subtyping-ATRT", "results")
+
+if (!dir.exists(results_dir)) {
+  dir.create(results_dir)
+}
+
+# Read in metadata
+metadata <-
+  readr::read_tsv(file.path(root_dir, "data", "pbta-histologies.tsv"))
+
+# Select wanted columns in metadata for merging and assign to a new object
+select_metadata <- metadata %>%
+  dplyr::select(sample_id,
+                Kids_First_Biospecimen_ID,
+                tumor_ploidy)
+
+# Read in gistic broad value data 
+gistic_focal_data <-
+  data.table::fread(
+    file.path(
+      root_dir,
+      "analyses",
+      "molecular-subtyping-ATRT",
+      "data",
+      "focal_data_by_genes.csv"
+    )
+  )
+
+# Read in ATRT subset focal CN data
+cn_df <- readr::read_tsv(
+  file.path(
+    root_dir,
+    "analyses",
+    "molecular-subtyping-ATRT",
+    "atrt-subset",
+    "atrt_focal_cn.tsv.gz"
+  )
+)
+
+# Read in final output from `01-ATRT-molecular-subtyping-data-prep.Rmd`
+final_df <-
+  data.table::fread(
+    file.path(
+      root_dir,
+      "analyses",
+      "molecular-subtyping-ATRT",
+      "results",
+      "ATRT_molecular_subtypes.tsv"
+    )
+  )
+```
+
+# Filter GISTIC data for ATRT samples
```

Comment:
A lot is happening in this section, can you add some more comments as to the requirements of what you are trying to end up with? I think we might be able to make this a tad easier to follow or more streamlined but its hard to say at first glance without my having looked at the previous notebooks.